example-model/src/main/java/uk/gov/gchq/palisade/example/configurator/ExampleConfigurator.java

```diff
@@ -22,10 +22,10 @@
 import org.springframework.cloud.client.ServiceInstance;
 import org.springframework.cloud.netflix.eureka.EurekaServiceInstance;
 
+import uk.gov.gchq.palisade.clients.simpleclient.request.AddSerialiserRequest;
```

Comment:
This should be a copy but will become irrelevant when the data-service uses the cache warming process so happy to leave for now.